doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -180,13 +180,31 @@ Provisioning registration data onto device
 The Fast Pair standard requires provisioning the device with Model ID and Anti-Spoofing Private Key obtained during device model registration.
 In the |NCS|, the provisioning data is generated as a hexadecimal file using the :ref:`bt_fast_pair_provision_script`.
 
-When building the Fast Pair in the |NCS|, the build system automatically calls the Fast Pair provision script and includes the resulting hexadecimal file in the firmware (the :file:`merged.hex` file).
+When building Fast Pair in the |NCS|, the build system automatically calls the Fast Pair provision script.
+It then includes the resulting hexadecimal file in the final firmware that you can flash onto the device.
+The Fast Pair provisioning data is stored on the dedicated Fast Pair partition, which has to be defined.
+
+Partition definition using the Partition Manager (PM)
+-----------------------------------------------------
+For devices that support :ref:`partition_manager` and have the :kconfig:option:`CONFIG_PM_SINGLE_IMAGE` Kconfig option automatically enabled by the Fast Pair build system, the system also automatically creates the ``bt_fast_pair`` partition.
+The partition is defined in the :file:`subsys/partition_manager/pm.yml.bt_fast_pair` file.
+The :ref:`fast_pair_input_device` sample follows this approach.
+Alternatively, the Fast Pair partition can be defined manually in the application's configuration file.
+To see how to do this, refer to the example in the :file:`samples/bluetooth/fast_pair/locator_tag/configuration/pm_static_nrf52840dk_nrf52840.yml` file which is a part of the :ref:`fast_pair_locator_tag` sample.
+For more information about defining Partition Manager partitions, see the :ref:`Configuration <pm_configuration>` section of the :ref:`partition_manager` page.
+
+Partition definition using the Devicetree (DTS)
+-----------------------------------------------
+For devices that don't support :ref:`partition_manager`, you must declare the ``bt_fast_pair_partition`` partition manually in the devicetree.
+Currently the :ref:`zephyr:nrf54h20dk_nrf54h20` board is the only device that requires manual partition definition.
```

Comment:
Wouldn't write that as we will have to maintain it with every new board. Maybe change that nrf54h20 is an example of SoC which uses it

Reply:
I'd leave it as is and remove this sentence completely once new target without partition manager support appears.

Reply:
We have this line of code: https://github.com/nrfconnect/sdk-nrf/blob/81d8a9e19b6a1a8e4eb1e06d97625b09619bc4ce/subsys/bluetooth/services/fast_pair/Kconfig.fast_pair#L134 so it has to be maintained anyway.

Reply:
ok :)